From fa7c87bc676fcdcf81853debf3bcbe4788d26e0a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:16 -0400
Subject: [redhat] kabi: Adding symbol __put_user_8 (arch/x86/lib/putuser.S)

Message-id: <03d9a8fc445fcf0cb492bedfab109eff0cf5a877.1410510773.git.poros@redhat.com>
Patchwork-id: 95892
O-Subject: [PATCH RHEL7.1 016/233] kabi: Adding symbol __put_user_8 (arch/x86/lib/putuser.S)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Not referenced directly, gets introduced due to inline helper
    function put_user. put_user is used to write a simple value to userspace.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__put_user_8

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__put_user_8 b/redhat/kabi/kabi-module/kabi_x86_64/__put_user_8
new file mode 100644
index 0000000..0b5ef4c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__put_user_8
@@ -0,0 +1,2 @@
+#1-
+0xb8e7ce2c __put_user_8 vmlinux EXPORT_SYMBOL
-- 
1.7.1